### PR TITLE
Remove dependency of addon build pipelines from release pipeline

### PR DESCRIPTION
**Pull request by kritika-singh3**
- removed task related to promoting addons from exp to stable
 - removed task related to bumping up extension docs

### CODE REVIEW

**Comment by kritika-singh3** [comment]
For now - I have just removed the stages/jobs which does the necessary. Post release we can clean up the tasks as well.

cc:\ @maheshp

File: .gocd/build.gocd.groovy [hunk 1]
```diff
@@ -511,18 +511,6 @@ GoCD.script {
...
         }
-        dependency('upload-addons') {
-          pipeline = 'upload-addons'
-          stage = 'upload-addons'
-        }
-        dependency('go-packages') {
-          pipeline = 'go-packages'
-          stage = 'fetch_from_build_go_cd'
-        }
-        dependency('go-addon-build') {
```
Comment (maheshp) [inline .gocd/build.gocd.groovy:522]: We need to remove `enterprise` material as well right?

Comment (kritika-singh3) [reply .gocd/build.gocd.groovy:522]: Yes, sorry missed that.

Comment (kritika-singh3) [reply .gocd/build.gocd.groovy:522]: It needs to be updated to `gocd` repo right?

Comment (maheshp) [reply .gocd/build.gocd.groovy:522]: I don't see a need to add `gocd` as material. Let's talk later.

**Pull request merged**